tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -120,7 +120,7 @@ class KZG {
       return;
     }
 #endif
-    cpu_batch_commitments_.resize(size);
+    cpu_batch_commitments_.resize(size, Bucket::Zero());
```

Comment:
It seems like we don't need to initialize the values, since `cpu_batch_commitments_` will be filled with the values. But just add a comment that why this is safe.